firebase.go

```diff
@@ -105,6 +105,22 @@ func (fb *Firebase) Unauth() {
 	fb.params.Del(authParam)
 }
 
+// Ref returns a copy of an existing Firebase reference with a new path.
+func (fb *Firebase) Ref(path string) *Firebase {
+	newFB := fb.copy()
+	parsedURL, err := _url.Parse(fb.url)
+	if err != nil {
+		panic(err)
```

Comment:
Okay, just fixed that, I only chose panic because it's so rare that the url will be unable to be parsed and it keeps it simple by only returning a single argument. But I agree that an error being returned could be better than panicing the whole program.